taos-optin/src/raw/mod.rs

```diff
@@ -93,6 +93,7 @@ pub struct ApiEntry {
 
     // query
     taos_query: unsafe extern "C" fn(taos: *mut TAOS, sql: *const c_char) -> *mut TAOS_RES,
+    taos_query_with_reqid: unsafe extern "C" fn(taos: *mut TAOS, sql: *const c_char, req_id: u64) -> *mut TAOS_RES,
```

Comment:
Use `Option<>` like signature for a method not compatible to all versions.

Reply:
done!